R/importMothur.R

```diff
@@ -63,66 +63,66 @@ NULL
 #' @importFrom SummarizedExperiment SummarizedExperiment
 #' @importFrom S4Vectors make_zero_col_DFrame
 #' @export
-importMothur <- function(sharedFile,
-                            taxonomyFile = NULL,
-                            designFile = NULL) {
+importMothur <- function(assay.file,
+                            row.file = NULL,
+                            col.file = NULL) {
 
```

Comment:
You did not deprecate old names

Reply:
Also they are missing from documentation (the old names)

Reply:
when i deprecate old name here, function is unable to locate file.